### ISSUE TITLE
Instrumentation logging level ignored

### ISSUE THREAD

**Issue by multicaststorm** [microsoft/ApplicationInsights-Java#1415]
### Expected behavior
When set the following in applicationinsights.json, I expect to turn off auto log collection (JUL in my case):
```
{
  "instrumentation": {
    "logging": {
      "level": "OFF"
    }
  }
}
```

### Actual behavior
I am still seeing log traces appear in azure monitor

### To Reproduce
Set config as described above and check if trace logs are being sent

**Comment by multicaststorm** [comment]
From a quick scan through the code, I think that it is expecting 'threshold' instead of 'level'
```
{
  "instrumentation": {
    "logging": {
      "threshold": "OFF"
    }
  }
}
```
...so either the code or the documentation is incorrect

**Comment by trask** [comment]
oh no, thanks!! will be fixed in next release by #1416
